content/guides/get-started/get-docker-desktop.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Noting that this page was moved to manuals recently. https://github.com/docker/docs/commit/951147d39fc886a7a0673db63ae83d0e281e2a3d
I see that it is more incorporated to the guide here, but it now exists in multiple places.